app/controllers/ingestibles_controller.rb

```diff
@@ -50,13 +50,18 @@ def create
   # PATCH/PUT /ingestibles/1 or /ingestibles/1.json
   def update
     if @ingestible.update(ingestible_params)
-      flash.now.notice = t('.success')
-      render :edit
+      redirect_to edit_ingestible_url(@ingestible), notice: t('.success')
     else
       render :edit, status: :unprocessable_entity
     end
   end
 
+  def update_markdown
```

Comment:
Added separate action to update markdown.
We can move markdown parsing code here instead of calling it on every edit request